github/oauth_server.go

```diff
@@ -13,82 +17,107 @@ import (
 )
 
 var (
-	ServerBind    = "127.0.0.1:7024"
-	ServerAddress = "http://127.0.0.1:7024"
-)
-
-var (
-	oauthConf = oauth2.Config{
-		ClientID:     "d5a2938d576279fbc995",
-		ClientSecret: "aadcab62248629c3d0018b9e54e4d3fc37fdd29e",
-		Scopes:       []string{"user:email", "repo"},
-		Endpoint:     githuboauth.Endpoint,
-	}
+	oauthClientId       = "d5a2938d576279fbc995"
```

Comment:
I can't think of any material benefit, but is it worth pulling this up to a const or something, rather than inlining it?

Reply:
So something like this?

```diff
diff --git a/github/oauth_server.go b/github/oauth_server.go
index 73c3130..f8f172e 100644
--- a/github/oauth_server.go
+++ b/github/oauth_server.go
@@ -16,7 +16,7 @@
        "golang.org/x/oauth2"
 )
 
-var (
+const (
        oauthClientId       = "d5a2938d576279fbc995"
        oauthScopes         = "user:email,repo"
        githubStageOneUri   = "https://github.com/login/device/code"
```